src/components/HardwareListing/HardwareListing.tsx

```diff
@@ -100,7 +100,12 @@ const ModuleGroup = (props: { level: number; group: ListingGroup; hideGroup?: bo
               props.group.groups[a].name.localeCompare(props.group.groups[b].name),
           )
           .map((groupId) => {
-            return <ModuleGroup key={groupId} level={props.level + 1} group={props.group.groups[groupId]} />;
+            return <ModuleGroup
+              key={groupId}
+              level={props.hideGroup ? props.level : props.level + 1}
+              group={props.group.groups[groupId]}
+              hideGroup={props.hideGroup ? props.hideGroup - 1 : 0}
```

Comment:
```suggestion
              hideGroup={props.hideGroup !== 0 ? props.hideGroup - 1 : 0}
```

Being more exact that we expect a number here, makes it a bit easier to read. But really, nitpicking :)

Same on line 114: `if (props.hideGroup !== 0)` :)

Reply:
I like the simple check as it also supports passing true/false and the original way will still work as expected :)
But yes, `ModuleGroup` is not exported so it would be safe to make this change.